basic_network_test.go

```diff
@@ -1138,53 +1133,17 @@ func TestMasterPlaylistIntegration(t *testing.T) {
 	if err != nil {
 		t.Errorf("Error getting master playlist: %v", err)
 	}
-	timer := time.NewTimer(time.Second * 3)
```

Comment:
Is there any difference between using this timer's channel vs. using `time.After` in the change?

Reply:
`time.After` is a more concise way than timer.  I learned this a few months ago reading the Lightning codebase.